application/versions.txt

```diff
@@ -2,28 +2,28 @@ adldap2/adldap2                           v10.5.0            A PHP LDAP Package
 behat/behat                               v3.13.0            Scenario-oriented BDD framework for PHP
 codeception/codeception                   5.0.10             BDD-style testing framework
 codeception/module-asserts                2.0.1              Codeception module containing various assertions
-codeception/module-phpbrowser             3.0.0              Codeception module for testing web application o...
+codeception/module-phpbrowser             3.0.0              Codeception module for testing web application over HTTP
```

Comment:
Not that it's important, but you can get the new text to match the old by changing the width of your terminal window.